train.py

```diff
@@ -0,0 +1,160 @@
+
+import numpy as np
+import torch
+import time
+import os
+import random
+
+from torch.utils.data import DataLoader
+from torchvision.models import mobilenet_v2
+from torchvision import transforms
+from torch import nn
+
+from triplettorch import HardNegativeTripletMiner
+from triplettorch import AllTripletMiner
+from torch.utils.data import DataLoader
+from triplettorch import TripletDataset
+
+from torchvision import transforms
+from torchvision import datasets
+import matplotlib.pyplot as plt
+
+from pdd.train_test_split import datadir_train_test_split
+from pdd.data_utils import AllCropsDataset
+from pdd.model import PDDModel
+from pdd.trainer import TripletTrainer
+from pdd.metrics import knn_acc
+# from train_test_split import datadir_train_test_split
+# from data_utils import AllCropsDataset
+# from model import PDDModel
+# from trainer import TripletTrainer
+
+RANDOM_SEED=13
+CUDN_DETERMENISTIC=True
+NUM_CLASSES=15
+
+# DATA_PATH = 'pdd'
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+DATA_ZIP_PATH = 'archive_full.zip'
+DATA_PATH = 'data/'
+TEST_SIZE = 0.2
+N_SAMPLE= 8
+BATCH_SIZE= 32
+EPOCHS=5000
+
+
+
+def fix_random_seed(seed, cudnn_determenistic=False):
+    random.seed(seed);
+    np.random.seed(seed)
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed(seed)
+    torch.backends.cudnn.deterministic=cudnn_determenistic
+
+def unzip_data():
+    os.system("unzip %s -d %s" % (DATA_ZIP_PATH, DATA_PATH)) 
+
+
+
+def prepare_datasets():
+    train_ds = AllCropsDataset(
+        DATA_PATH, 
+        subset='train',
+        transform=transforms.Compose([
+            transforms.Resize(256),
```

Comment:
all transforms move to the arguments like `train_transform` and `test_transform`